components/style/gecko_conversions.rs

```diff
@@ -6,10 +6,41 @@
 //! Ideally, it would be in geckolib itself, but coherence
 //! forces us to keep the traits and implementations here
 
+#![allow(unsafe_code)]
+
 use app_units::Au;
+use gecko_bindings::bindings::{RawServoStyleSheetStrong, ServoComputedValuesStrong};
 use gecko_bindings::structs::nsStyleCoord_CalcValue;
+use gecko_bindings::sugar::refptr::HasStrong;
+use properties::ComputedValues;
+use std::mem::transmute;
+use std::ptr::null;
+use std::sync::Arc;
+use stylesheets::Stylesheet;
 use values::computed::{CalcLengthOrPercentage, LengthOrPercentage};
 
+macro_rules! unsafe_impl_strong {
+    ($strong:ty, $servo:ty) => {
+        unsafe impl HasStrong for $servo {
```

Comment:
Shouldn't this trait be named IntoStrong rather than HasStrong, to match the name of its method?